card/mod/03_machines/lib/card/machine.rb

```diff
@@ -49,12 +49,12 @@ def store_machine_output args={}, &block
 
     def self.included(host_class)
       host_class.extend( ClassMethods )
-      host_class.output_config = { :filetype => "txt" }
+      host_class.output_config = { filetype: "txt" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.